### DISCUSSION TITLE
What is the purpose of `index.norg` and `inbox.norg`?

### DISCUSSION THREAD

**Discussion by DennisTheMenace780** [nvim-neorg/neorg#686] [General]
I am just getting acclimated to Neorg but I have noticed these two particular files seem to get autogenerated quite a bit. I have been trying to read the documentation to better understand what these files do, but it's not overly clear to me what purpose they are intended to serve. 

Maybe I missed something as I was skimming, but I would love to know so that I can better utilize them in my workflow.

**Comment by max397574** [comment]
inbox.norg is something for neorg-gtd (there is a wiki page and a help page on that)
index.norg is just an index file used by many people where they put links to other files in a folder
otherwise it isn't anything special
